{
  "Faults": [
    {
      "latitude": 51.764862919050415,
      "type": "LV",
      "reference": "KX6317",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-24T21:20:00.000+0000",
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0c06e6bc-f1f0-4dcb-ac37-0e592c18492c",
      "minLongitude": -1.1937,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1920108508566918,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-25T00:52:00.000+0000",
      "estimatedRestoration": "2023-05-25T13:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6317'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6317",
      "maxLongitude": -1.1895,
      "minLatitude": 51.7638,
      "name": "OX3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX3 8BN",
        "OX3 8BW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1937,
              51.7666
            ],
            [
              -1.1895,
              51.7666
            ],
            [
              -1.1895,
              51.7638
            ],
            [
              -1.1937,
              51.7638
            ],
            [
              -1.1937,
              51.7666
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-24T21:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7666,
      "etrUpdate": true
    },
    {
      "latitude": 51.305133983763874,
      "type": "LV",
      "reference": "KX6346",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1eab0f6b-c621-46e0-b471-8f9ec1ca8727",
      "minLongitude": -0.7237,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.719415387128518,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-24T21:34:00.000+0000",
      "estimatedRestoration": "2023-05-25T12:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6346'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6346",
      "maxLongitude": -0.7138,
      "minLatitude": 51.303,
      "name": "GU16 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU16 6QA",
        "GU16 6QB",
        "GU16 6RB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7237,
              51.3072
            ],
            [
              -0.7138,
              51.3072
            ],
            [
              -0.7138,
              51.303
            ],
            [
              -0.7237,
              51.303
            ],
            [
              -0.7237,
              51.3072
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-24T22:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3072,
      "etrUpdate": true
    },
    {
      "latitude": 50.62799715878681,
      "type": "LV",
      "reference": "KX6401",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-25T01:15:00.000+0000",
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "022c5c27-8251-45be-bc90-63cbe17edcb3",
      "minLongitude": -2.4894,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.4790014136233993,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-24T23:58:00.000+0000",
      "estimatedRestoration": "2023-05-25T12:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6401'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6401",
      "maxLongitude": -2.4761,
      "minLatitude": 50.6259,
      "name": "DT4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT4 9RR",
        "DT4 9RW",
        "DT4 9RZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.4894,
              50.6301
            ],
            [
              -2.4761,
              50.6301
            ],
            [
              -2.4761,
              50.6259
            ],
            [
              -2.4894,
              50.6259
            ],
            [
              -2.4894,
              50.6301
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-25T01:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.6301,
      "etrUpdate": true
    },
    {
      "latitude": 51.549526663615524,
      "type": "LV",
      "reference": "KX6453",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-05-25T05:00:00.000+0000",
      "affectedCustomerCount": 19,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "2a45b7ef-d288-49b1-a2ee-419456572ec3",
      "minLongitude": -0.4776,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.47614803644269044,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-25T03:40:00.000+0000",
      "estimatedRestoration": "2023-05-25T07:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6453'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6453",
      "maxLongitude": -0.4743,
      "minLatitude": 51.549,
      "name": "UB8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "UB8 1AU",
        "UB8 1AX",
        "UB8 1DB",
        "UB8 1DG",
        "UB8 1DQ",
        "UB8 1QS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.4776,
              51.5507
            ],
            [
              -0.4743,
              51.5507
            ],
            [
              -0.4743,
              51.549
            ],
            [
              -0.4776,
              51.549
            ],
            [
              -0.4776,
              51.5507
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-25T04:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5507,
      "etrUpdate": true
    },
    {
      "latitude": 50.77710905174705,
      "type": "HV",
      "reference": "KX6449",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a1c4b7a7-3288-4c37-a60c-9e49f351ed40",
      "minLongitude": -2.4017,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.394135053427672,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-25T03:13:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on their way to site to get the power back on as quickly as they can. please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6449'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6449",
      "maxLongitude": -2.3871,
      "minLatitude": 50.7721,
      "name": "DT2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT2 7TU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.4017,
              50.7805
            ],
            [
              -2.3871,
              50.7805
            ],
            [
              -2.3871,
              50.7721
            ],
            [
              -2.4017,
              50.7721
            ],
            [
              -2.4017,
              50.7805
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-25T04:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7805,
      "etrUpdate": true
    },
    {
      "latitude": 51.54401034977589,
      "type": "LV",
      "reference": "KX6487",
      "etaUpdate": true,
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "24bc7b2b-2838-42e2-adc6-9d8a9cce5eac",
      "minLongitude": -0.899,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8842568094539105,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-25T05:15:00.000+0000",
      "estimatedRestoration": "2023-05-25T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6487'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6487",
      "maxLongitude": -0.833,
      "minLatitude": 51.5233,
      "name": "RG9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG9 2LQ",
        "RG9 2LT",
        "RG9 2LX",
        "RG9 3HH",
        "RG9 3HX",
        "RG9 3JD",
        "RG9 6PS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.899,
              51.6069
            ],
            [
              -0.833,
              51.6069
            ],
            [
              -0.833,
              51.5233
            ],
            [
              -0.899,
              51.5233
            ],
            [
              -0.899,
              51.6069
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.6069,
      "etrUpdate": true
    },
    {
      "latitude": 51.53914536125409,
      "type": "LV",
      "reference": "KX6411",
      "etaUpdate": true,
      "affectedCustomerCount": 32,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "913ea2a5-44ea-4672-99ad-f6c41d9a94dc",
      "minLongitude": -0.7393,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7328009782263104,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-05-25T00:34:00.000+0000",
      "estimatedRestoration": "2023-05-25T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KX6411'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KX6411",
      "maxLongitude": -0.7223,
      "minLatitude": 51.5277,
      "name": "SL6 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL6 7LY",
        "SL6 7PZ",
        "SL6 7QF",
        "SL6 7QG",
        "SL6 7QH",
        "SL6 7QJ",
        "SL6 7QL",
        "SL6 7QQ",
        "SL6 7QX",
        "SL6 7RH",
        "SL6 7TX",
        "SL6 7UA",
        "SL6 7UE",
        "SL6 7UF",
        "SL6 7UP",
        "SL6 7UT",
        "SL6 7UU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7393,
              51.5452
            ],
            [
              -0.7223,
              51.5452
            ],
            [
              -0.7223,
              51.5277
            ],
            [
              -0.7393,
              51.5277
            ],
            [
              -0.7393,
              51.5452
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-05-25T04:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5452,
      "etrUpdate": true
    }
  ]
}
